resources/report/plotting/uv_17lev.ct

```diff
@@ -0,0 +1,23 @@
+; Type: COSMO Library Color Table
+; Description: Wind speed representation in 17 levels. This color table
+;   is used in the model browser to plot wind at 10m, 850, 700 and 500hPa
+17
+4.0 6.0 10.0 14.0 18.0 22.0 26.0 30.0 35.0 40.0 45.0 50.0 60.0 70.0 80.0 90.0 100.0
```

Comment:
range seems quite large here, especially for surface winds, can we double check?

Reply:
most other plots I've seen from anemoi use a max range of 25 m/s (about 50 knots) for wind...